recipes/yt/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py --quiet install --single-version-externally-managed --record=record.txt
```

Comment:
I was noticing on my windows VM that `conda build` was using the line in the meta.yaml file, not the `bld.bat` file. I ended up recreating the separate build scripts, so I could customize the behavior on windows.